code/src/app/dashboard/editcar/page.tsx

```diff
@@ -1,24 +1,25 @@
+import EditCar from '@/components/EditCar';
 import DashboardLayout from '../../../components/DashboardLayout';
-import EditCar from '../../../components/EditCar';
 
 export default function page() {
   return (
     <DashboardLayout>
-      <div>
-        <EditCar
-          carId="2"
-          manufacturer="Toyota"
-          type="Sedan"
-          year={2022}
-          license="ABC-123"
-          mileage={10000}
-          model="Camry"
-          color="Red"
-          seats={5}
-          condition="New"
-          oilChange="2022-01-01"
-        />
-      </div>
+      <EditCar
+        carId="1"
+        manufacturer="Toyota"
+        type="Sedan"
+        year={2022}
+        license="ABC-123"
+        mileage={10000}
+        model="Camry"
+        color="Red"
+        seats={5}
+        condition="New"
+        mileageLastOilChange={5000}
+        mileageLastTireChange={10000}
+        dateNextTireChange={new Date('2023-01-01')}
```

Comment:
I think my endpoint uses a string, does it not work with string objects?

Reply:
ill check it

Reply:
@ncruz93 Nick What I saw from your endpoint https://github.com/BUMETCS673/project-team-6/blob/add-car-endpoint/code/src/models/Car.js at lane 45 and 46, it is Date type. let me know if you changed to String, I can do that too

Reply:
I think something like `(new Date('2023-01-01')).toISOString()`?